independent-publisher-connectors/Coupa/readme.md

```diff
@@ -0,0 +1,237 @@
+# Coupa
+See all of your business spend in one place with Coupa to make cost control, compliance and anything spend management related easier and more effective.
+This connector provides robust access to read, edit, or integrate your data with the Coupa platform
+
+## Publisher: NovaGL
+
+## Prerequisites
+
+Login to your Coupa instance with an administrator\integration administrator account
```

Comment:
@NovaGL can you please provide some clear steps on how to get the client credentials.?
Please provide some links. Thank you so much.

Reply:
Can you be more clear. What is missing.

After they fill in the below details the keys pop up on the screen.

Reply:
@NovaGL Please provide steps to get the credentials.

Reply:
Do you mean credentials to login to coupa?

Reply:
Screenshot for credentials screen added hopefully this resolves this,

Reply:
Please provide a link to this page 
![image](https://user-images.githubusercontent.com/97551789/213051885-f5a7fb2f-4304-4875-868e-6cd806dcb4a1.png) and clear steps on how to create an account with Coupa. Thank you